ios/brave-ios/scripts/bootstrap.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/sh
+# Copyright (c) 2021 The Brave Authors. All rights reserved.
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this file,
+# You can obtain one at https://mozilla.org/MPL/2.0/.
+
+set -e
+
+this_dir=$(dirname $0)
+src_dir=$(realpath "$this_dir/../../../..")
+
+# Initial packing of JS so SPM doesnt show warnings that the files are missing
+# Subsequent builds will run this to ensure those files are kept up to date
+npm run ios_pack_js
```

Comment:
unrelated: I wonder if we should have `npm audit` somewhere to remind us of package vulns

Reply:
I believe that will run automatically when you run `npm run sync ...` commands